collect_app/src/main/java/org/odk/collect/android/preferences/GeneralSharedPreferences.java

```diff
@@ -74,7 +73,7 @@ public void reset(String key) {
     }
 
     public GeneralSharedPreferences save(String key, Object value) {
-        editor = sharedPreferences.edit();
+        android.content.SharedPreferences.Editor editor = sharedPreferences.edit();
```

Comment:
Do you prefer fully qualifying the type here, as opposed to importing it?

Reply:
No, I don't. Fixed.